yt/visualization/plot_modifications.py

```diff
@@ -1643,7 +1645,11 @@ def __call__(self, plot):
         y0, y1 = plot.ylim
         xx0, xx1 = plot._axes.get_xlim()
         yy0, yy1 = plot._axes.get_ylim()
-        reg = self._get_region((x0,x1), (y0,y1), plot.data.axis, data)
+        if type(self.data_source)==YTCutRegion:
+            mylog.warn("Parameter 'width' is ignored in annotate_particles if the data_source is a cut_region.")
```

Comment:
I will try to look closer at this, unfortunately I'm totally buried under other yt issues :/

Maybe add a link to the github issue tracking this problem in the error text?